### PR TITLE
feat(suite): btn to dump reducer's data from debug settings

### LINKED ISSUE #3719: Dump the content of reducers
into a file or some textarea to it can be copy pasted.

In first iteration we could hide it in debug settings, so only QA will use this and we don't have to bother with redacting sensitive data.

my thoughts
https://github.com/trezor/trezor-suite/issues/239#issuecomment-762968914

### PR DESCRIPTION
**Pull request by slowbackspace**
close https://github.com/trezor/trezor-suite/issues/3719

no data censoring, just for QA purposes (that's why it is hidden in debug settings).
I would really like to see this as a public facing feature after implementing bulletproof censoring of sensitive data (balances, addresses, txids, xpubs,...).

### CODE REVIEW

File: packages/suite/src/actions/suite/suiteActions.ts [hunk 1]
```diff
@@ -78,6 +79,12 @@ export type SuiteAction =
           colors: SuiteThemeColors;
       };
 
+export const dumpReducers = () => (_dispatch: Dispatch, getState: GetState) => {
+    const b = new Blob([JSON.stringify(getState(), null, 2)]);
+    const timestamp = Math.floor(new Date().getTime() / 1000);
+    saveAs(b, `suite_reducers_${process.env.VERSION}_${timestamp}.txt`);
```
Comment (keraf) [inline packages/suite/src/actions/suite/suiteActions.ts]: Should probably be saved as `.json`. And maybe worth adding a content header as well.

**Review by keraf — CHANGES_REQUESTED** [review]
I think this approach is alright but could be better. Ideally we should have a QA/tools overlay to do this at any point in the application, without having to navigate away from what we were doing.

**Comment by tomexx** [comment]
We discussed it in QA Slack channel and they can simply run `JSON.stringify(__NEXT_REDUX_STORE__.getState())` directly from console (we should document it somewhere maybe).

But I think we still can proceed with Maros' PR and make the dump tool more accessible later on.

**Comment by matejkriz** [comment]
feel free to reopen...

**Pull request closed without merging**